openedx/core/djangoapps/user_api/accounts/image_helpers.py

```diff
@@ -22,12 +23,35 @@
 
 def get_profile_image_storage():
     """
-    Configures and returns a django Storage instance that can be used
-    to physically locate, read and write profile images.
+    Returns an instance of the configured Django storage class for profile images.
+
+    The function looks for the `PROFILE_IMAGE_BACKEND` setting in the Django settings.
+    If it exists and includes a `'class'` key, that class is used as the storage backend.
+    If the setting is missing or does not include a `'class'`, the default storage backend
+    DEFAULT_FILE_STORAGE will return.
+
+    Returns:
+        An instance of the configured storage backend.
     """
-    config = settings.PROFILE_IMAGE_BACKEND
-    storage_class = get_storage_class(config['class'])
-    return storage_class(**config['options'])
+    config = getattr(settings, 'PROFILE_IMAGE_BACKEND', {})
+    storage_class_path = config.get('class')
+    options = config.get('options', {})
+
+    if not storage_class_path:
+        storages_config = getattr(settings, 'STORAGES', {})
+
+        if 'PROFILE_IMAGE_STORAGE' in storages_config:
```

Comment:
Note: This implementation is forward-compatible and is intended to work when the legacy PROFILE_IMAGE_BACKEND setting is removed and the new Django STORAGES configuration is used instead. For example:
```
STORAGES = {
    'PROFILE_IMAGE_STORAGE': {
        'BACKEND': 'storages.backends.s3boto3.S3Boto3Storage',
        'OPTIONS': {
            'bucket_name': 'profiles',
            'default_acl': 'public',
            'location': 'profile/images',
        }
    }
}
```